tests/test_sh_class.py

```diff
@@ -0,0 +1,128 @@
+"""
+Tests for spherical harmonic class
+"""
+import pytest
+import numpy as np
+import pyfar as pf
+from spharpy.spherical import SphericalHarmonics
+from spharpy.samplings import gaussian, calculate_sampling_weights
+
+def test_sphharm_init():
+    coordinates = pf.Coordinates(1, 0, 0)
+    sph_harm = SphericalHarmonics(n_max=2, coordinates=coordinates)
+    assert sph_harm.n_max == 2
+    assert np.all(sph_harm.coordinates == coordinates)
```

Comment:
I would suggest to test the values of the optional parameters at least once as well, e.g., `sph_harm.basis_type == 'real'`